qmlt/helpers.py

```diff
@@ -41,26 +41,26 @@
 import numpy as np
 
 
-def sample_from_distr(distr):
+def sample_from_distribution(distribution):
```

Comment:
I think I am okay with this change - it's always a trade off between concise function names and descriptive function names, but this doesn't seem too long. @mariaschuld, your thoughts?

Reminder to update any other locations in this documentation that might be affected by this change.